### PR TITLE
[Fogbugz 1335732] Documentation and defaults for DLSS

### PR DESCRIPTION
**Pull request by kecho**
### Purpose of this PR
* Setting up correct defaults for DLSS, current defaults are bad for UX because require more complexity to setup.
* Improve the DLSS guide, which is missing a couple of steps. Also linking to the Dynamic Resolution Guide.

[Fogbugz](https://fogbugz.unity3d.com/f/cases/1339128/)
---
### Testing status
* Run DLSS locally on windows with a brand new HDRP asset. Ensured that DLSS Use Optimal Settings is enabled by default.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md [hunk 1]
```diff
@@ -44,8 +44,14 @@ After you install the NVIDIA package, more properties appear in [HDRP Assets](HD
 Your Unity project now supports DLSS and you can now enable DLSS for Cameras in your scene. Enabling DLSS in the HDRP Asset exposes other properties that you can use to customize DLSS. For information about these properties, see the [HDRP Asset](HDRP-Asset.md) documentation.
 
 1. In the Hierarchy or Scene view, select a Camera and view it in the Inspector.
-2. Select **Allow DLSS**.
-3. Enabling **Allow DLSS** exposes other properties that you can use to customize DLSS for the Camera. For information about these properties, see the [Camera](HDRP-Camera.md) documentation.
+2. Select **Allow Dynamic Resolution** to expose the DLSS settings. For more information see the [Dynamic Resolution](Dynamic-Resolution.md) guide. 
+3. Select **Allow DLSS**.
+4. Enabling **Allow DLSS** exposes other properties that you can use to customize DLSS for the Camera. For information about these properties, see the [Camera](HDRP-Camera.md) documentation.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md:49]: Remove passive voice: 

Enable **Allow DLSS** to expose other properties that you can use to customize DLSS for the Camera.

File: com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md [hunk 2]
```diff
@@ -44,8 +44,14 @@ After you install the NVIDIA package, more properties appear in [HDRP Assets](HD
...
+
+### DLSS and Dynamic Resolution
+
+By default the [HDRP Assets](HDRP-Asset.md) has the **Use Optimal Settings** checkbox enabled. This means that DLSS will try and set the dynamic resolution scale automatically. If this checkbox is disabled,
+DLSS will utilize the same dynamic resolution scale set by the project. For more information see the [Dynamic Resolution](Dynamic-Resolution.md) guide.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md:55]: Some small tweaks: 

The **Use Optimal Settings** checkbox in the [HDRP Asset](HDRP-Asset.md) is enabled by default. This means that DLSS sets the dynamic resolution scale automatically. If you disable this checkbox DLSS uses the same dynamic resolution scale set by the project. For more information see the [Dynamic Resolution](Dynamic-Resolution.md) guide.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Couple of small edits :)

File: com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md (hunk 1 above)
Comment (kecho) [reply com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md:49]: Done! thank you.

File: com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md (hunk 2 above)
Comment (kecho) [reply com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md:55]: Done! thank you.

**Review by iM0ve — APPROVED** [review]
Testing sufficient. Looks all good.

**Comment by sebastienlagarde** [comment]
note to self: retarget to master when hd/bugfix is merge

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**